README.md

```diff
@@ -118,7 +118,7 @@ Finally, static fields are executed and applied.
 This decorators proposal uses the syntax of the previous Stage 2 decorators proposal. This means that:
 - Decorator expressions are restricted to a chain of variables, property access with `.` but not `[]`, and calls `()`. To use an arbitrary expression as a decorator, `@(expression)` is an escape hatch.
 - Class expressions may be decorated, not just class declarations.
-- Class decorators come after `export` and `default`.
+- Class decorators may exclusively come before `export` or after `export [default]`.
```

Comment:
```suggestion
- Class decorators may exclusively come before, or after, `export`/`export default`.
```
the square bracket notation means "optional" in argument signatures and command line args but there's no convention for using it in code syntax, especially when brackets have a meaning already.

Reply:
good call